sqladmin/application.py

```diff
@@ -103,6 +113,22 @@ class UserAdmin(ModelAdmin, model=User):
 
         self._model_admins.append((model()))
 
+    def register_view(self, view: Type["ModelView"]) -> None:
+        class_view = view()
```

Comment:
I think this is the other way around, the class_view should be the argument because it is Type[...] and `view` should be instantiated from it, what do you think?